common/static/common/js/discussion/views/discussion_thread_list_view.js

```diff
@@ -153,11 +154,12 @@
                 });
             };
 
-            DiscussionThreadListView.prototype.addSearchAlert = function(message) {
+            DiscussionThreadListView.prototype.addSearchAlert = function(message, css_class) {
```

Comment:
Can you add a comment / [JSDoc](http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/javascript_guidelines.html#documentation) explaining the purpose of css_class? What's an example use case?